python_modules/dagster/dagster/_core/execution/context/invocation.py

```diff
@@ -607,6 +634,8 @@ def build_op_context(
     config: Any = None,
     partition_key: Optional[str] = None,
     mapping_key: Optional[str] = None,
+    run: Optional[DagsterRun] = None,
+    op_name: Optional[str] = None,
```

Comment:
I would expect to be able to derive `op_name` from the `op` once we bind the context to the invoked op